cluster/addons/addon-manager/README.md

```diff
@@ -40,20 +40,20 @@ The `addon-manager` is built for multiple architectures.
 ```console
 # Build for linux/amd64 (default)
 $ make push ARCH=amd64
-# ---> staging-k8s.gcr.io/addon-manager/kube-addon-manager-amd64:VERSION
```

Comment:
Are those changes merged into k/k's `/cluster'?

Reply:
As you mentioned `staging-k8s.gcr.io` need not be migrated. Thank you 👍